File: idea-plugin/src/main/java/org/ifinalframework/jetbrains/plugins/aio/api/yapi/model/CatMenu.java
package org.ifinalframework.jetbrains.plugins.aio.api.yapi.model;


import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Getter;
import lombok.Setter;


/**
 * 分类菜单
 *
 * @author iimik
 * @since 0.0.1
 **/
@Getter
@Setter
public class CatMenu {
    @JsonProperty("_id")
    private Long id;
    private String name;

    @Override
    public String toString() {
        return name;
    }
}

File: idea-plugin/src/main/kotlin/org/ifinalframework/jetbrains/plugins/aio/api/yapi/model/CatMenu.kt
package org.ifinalframework.jetbrains.plugins.aio.api.yapi.model

import com.fasterxml.jackson.annotation.JsonProperty


/**
 * 分类菜单
 *
 * @author iimik
 * @since 0.0.1
 **/
data class CatMenu(
    @JsonProperty("_id")
    val id: Long,
    val name: String,
) {
    override fun toString(): String {
        return name
    }
}
